news/2024/2024-07-28-the-perennial-concludes.md

```diff
@@ -0,0 +1,392 @@
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3958619">D I O</a></p>
+
+The Perennial gave almost full creative freedom to musicians this year; what kind of challenges does that present compared to a more rigid approach where musicians are given song specifications, and which approach do you enjoy more as a musician?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18983" style="background-image: url('/wiki/shared/news/2024-07-28-the-perennial-concludes/avatar-doomsday.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-fa" href="https://osu.ppy.sh/users/3958619">Doomsday</a></p>
+
+To be honest, I've not had much experience producing music set to rigid expectations for others (Unless using Vocaloid for Battle of the Box counts. I learned how to use it for *[Luyten's Star](https://osu.ppy.sh/beatmapsets/2134395#osu/4490753)*).
+
+I'd imagine that producing to more rigid constraints would be a challenge for me, as my creative process often involves trying lots of different ideas until one sounds good. I would find it tricky to work where I might not be able to settle on an idea that I like.
+
+That said, I've made a few tracks where I limit myself to certain sample sets, whether it's limited to Amiga samples or Touhou style instruments. In those instances, I gotta get creative with the composition side, which is where I have the most fun. Perhaps working within limits would fit me just fine!
+
+</div>
+
+---
+
+::{#chiv}::
+
+![](/wiki/shared/news/2024-07-28-the-perennial-concludes/chiv.png)
+
+[chiv](https://osu.ppy.sh/users/6701656) is a mappooler in a wide range of community tournaments big and small across multiple rank ranges; he is also one of the mappoolers for the Perennial.
+
+---
+
+<div class="news-chat-quote__text-container">
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/6701656" style="background-image: url('/wiki/shared/news/2024-07-28-the-perennial-concludes/avatar-chiv.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/6701656">chiv</a></p>
+
+Hi, I'm chiv. I'm currently part of the OWC mappooling team and I also work on Project Loved. During The Perennial, I was part of the mappooling staff, which focuses on creating fun, balanced and engaging mappools for players and viewers to enjoy.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3958619" style="background-image: url('/wiki/shared/news/2024-07-28-the-perennial-concludes/avatar-dio.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3958619">D I O</a></p>
+
+Before you were a mappooler, you were a tournament player; what made you decide to start mappooling rather than continuing to push for tournament wins?
```

Comment:
```suggestion
Before you were a mappooler, you were a tournament player. What made you decide to start mappooling rather than continuing to push for tournament wins?
```